tests/fixtures/node-cjs-importing-mjs/lib/file.mjs

```diff
@@ -0,0 +1,10 @@
+class App {
+  constructor(event, context) {
+    return {
```

Comment:
wait, what? Constructors can return something? this weird

Reply:
but fine for a fixture, I guess ^^

Reply:
The idea was to replicate [this scenario](https://stackoverflow.com/questions/66676555/nodejs-14-x-native-aws-lambda-import-export-support/69136438#69136438), but using a class or not is actually irrelevant for the test case here. All that matters is that the file is ESM.